File: FamilyTree/app/src/main/java/com/wanggang/familytree/model/FamilyDataBase.java
package com.wanggang.familytree.model;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {FamilyMemberEntity.class}, version = 1)
abstract public class FamilyDataBase extends RoomDatabase {

    public abstract FamilyMemberDao familyMemberDao();

}

File: FamilyTree/app/src/main/java/com/wanggang/familytree/model/FamilyDataBase.kt
package com.wanggang.familytree.model

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

@Database(entities = [FamilyMemberEntity::class], version = 1)
abstract class FamilyDataBase11: RoomDatabase() {

    abstract fun familyMemberDao(): FamilyMemberDao

}